data/mods/more_classes_scenarios/cs_scenarios.json

```diff
@@ -21,6 +21,17 @@
     "start_name" : "Mansion",
     "flags" : ["SUR_START"]
   },
+  {
+    "type": "scenario",
+    "ident": "last_crime",
+    "name": "The Last Jig",
```

Comment:
Not sure if `Jig` is intended for ["dance"](https://www.thefreedictionary.com/jig), or `Gig` as in [job](https://www.thefreedictionary.com/gig) [4th definition].

I can kinda see it going either way, but just wanted to make a note of it for potential clarification.

Reply:
Meant as Gig. Will fix.